ocaml/xapi/xapi_host.ml

```diff
@@ -2628,15 +2630,19 @@ let get_sched_gran ~__context ~self =
 
 let emergency_disable_tls_verification ~__context =
   (* NB: the tls-verification state on this host will no longer agree with state.db *)
+  let self = Helpers.get_localhost ~__context in
   Stunnel_client.set_verify_by_default false ;
-  Unixext.unlink_safe Xapi_globs.verify_certificates_path
+  Unixext.unlink_safe Xapi_globs.verify_certificates_path ;
+  Db.Host.set_tls_verification_enabled ~__context ~self ~value:false
 
 let emergency_reenable_tls_verification ~__context =
   (* NB: Should only be used after running emergency_disable_tls_verification.
      Xapi_pool.enable_tls_verification is not used because it introduces a
      dependency cycle. *)
+  let self = Helpers.get_localhost ~__context in
   Stunnel_client.set_verify_by_default true ;
-  Helpers.touch_file Xapi_globs.verify_certificates_path
+  Helpers.touch_file Xapi_globs.verify_certificates_path ;
+  Db.Host.set_tls_verification_enabled ~__context ~self ~value:false
```

Comment:
i think you mean `true` here?

Reply:
Good catch

Reply:
see my comment below, I think trying to modify the database in an emergency call will often not end well

Reply:
maybe, for the case when the slave will reconnect to the master after running the emergency call, a good place for this code is here: https://github.com/xapi-project/xen-api/blob/62e144780c6f1cccb17fc4decaa010365167b398/ocaml/xapi/xapi.ml#L1158

Reply:
So you would suggest a different strategy?  We could not update the database in the emergency command that disables TLS verification but we have to update the database at some point as otherwise there is no point to have this field.